index.html

```diff
@@ -201,6 +202,11 @@ <h1>Test the Apto PCI SDK</h1>
 				.addEventListener("click", () =>
 					AptoPCISdk.getIsDataVisible().then(console.info)
 				);
+			document
+				.getElementById("btnPIN")
+				.addEventListener("click", () =>
+					AptoPCISdk.showSetPinForm().then(console.info)
```

Comment:
This is not returning a `Promise` and hence not working locally, should we just remove the `then`?
```suggestion
					AptoPCISdk.showSetPinForm()
```